### PR TITLE
Converting the project to Zendesk

### PR DESCRIPTION
**Pull request by bcolferzd**
Converting the project to Zendesk by renaming ragurney personal reverences to zendesk.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
 FROM golang:1.11 as builder
 
-LABEL maintainer "Ryan Gurney <rygurney@zendesk.com>"
+LABEL maintainer "Brian Colfer <bcolfer@zendesk.com>"
```
Comment (dragonfax) [inline Dockerfile]: Maybe you want to put down the team for this instead?

File: Dockerfile [hunk 2]
```diff
@@ -13,16 +13,19 @@ RUN GIT_REV=$(git rev-parse HEAD) \
...
     && CGO_ENABLED=0 GOOS=linux \
 	go build -a -installsuffix cgo \
-	-o app -ldflags "-X main.GitRevision=${GIT_REV}" .
+	-o app -ldflags "-X main.GitRevision=${COMMIT_SHA}" .
 
-FROM scratch
+FROM debian:buster
 
-LABEL maintainer "Ryan Gurney <rygurney@zendesk.com>"
+LABEL maintainer "Build Release Engineering <bre@zendesk.com>"
```
Comment (dragonfax) [inline Dockerfile]: Is this redundant?

File: Dockerfile [hunk 3]
```diff
@@ -13,16 +13,19 @@ RUN GIT_REV=$(git rev-parse HEAD) \
...
+
+RUN mkdir /app
```
Comment (dragonfax) [inline Dockerfile]: Is this necessary? I thought docker creates the /app automatically?

File: Dockerfile (hunk 1 above)
Comment (bcolferzd) [reply Dockerfile]: The team is down below but I'll change this

File: Dockerfile (hunk 2 above)
Comment (bcolferzd) [reply Dockerfile]: Different image and will get this different maintainer label

Comment (bcolferzd) [reply Dockerfile]: Different image and will get this different maintainer label

File: Dockerfile (hunk 3 above)
Comment (bcolferzd) [reply Dockerfile]: No ... some minimal images have no directory

File: Dockerfile (hunk 1 above)
Comment (bcolferzd) [reply Dockerfile]: I set this to BRE

File: Dockerfile (hunk 2 above)
Comment (bcolferzd) [reply Dockerfile]: If I set the maintainer of the first image to BRE I think this label is redundant ... removing it

**Pull request closed without merging**